src/app/modules/image-viewer/image-viewer-legend/image-viewer-legend.component.ts

```diff
@@ -15,4 +15,22 @@ export class ImageViewerLegendComponent {
    * The list of layers used to render this components
    */
   @Input() layers: ImageViewerLayer[];
+
+  /**
+   * Determines whether the color icon is light enough to require a border (light mode only)
+   * @param color The color hex code
+   * @returns true if brightness is above a certain threshold
+   */
+  iconLight(color: string): boolean {
```

Comment:
Again these can be removed when is(Light|Dark) has been placed inside the ImageViewerLayer class